addons/mail/static/src/models/thread/thread.js

```diff
@@ -1401,6 +1401,22 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * Compute an url string that can be used inside a href attribute
+         *
+         * @private
+         * @returns {string}
+         */
+        _computeUrl() {
+            if (this.model !== 'mail.channel') {
+                const baseHREF = this.env.session.url('/web');
+                return `${baseHREF}#model=${this.model}&id=${this.id}`;
+            } else {
+                const baseHREF = this.env.session.url('/web');
```

Comment:
Same `baseHREF`, so can be before conditional.
Also would name it `baseHref` instead of `baseHREF` (easier to read).